### ISSUE TITLE
Suggestion: input masking with AlpineJS mask plugin

### ISSUE THREAD

**Issue by Visnicio** [mkocansey/bladewind#397]
Bladewind could have a mask attribute for inputs that if the user would want to use, he would need to add the AlpineJS mask plugin to the project, just like the date picker uses Alpine for its usability

**Comment by mkocansey** [comment]
Hi @Visnicio will what you are suggesting be different from `<x-bladewind::input type="password" viewable="true" />`?

**Comment by Visnicio** [comment]
I was thinking in something more like a date input, or telephone input, where you would use liek this
`<x-bladewind::input label="Telephone" mask="(99) 9999-9999">`

**Comment by mkocansey** [comment]
Oh right. I get you now. I will look into Alpine's mask plugin and see how easy that will be easy to integrate.

**Comment by mkocansey** [comment]
@Visnicio have you been able to use the Alpinejs masking plugin with the BladewindUI input? Doesn't seem to be working for me.

**Comment by mkocansey** [comment]
Yes. Please do and let me know if you managed to get it working.
